scripts/sanitize_db_dump/README.md

```diff
@@ -0,0 +1,62 @@
+# DB Dump Sanitizer
```

Comment:
suggestion (non-blocking): It might be worth pointing to this from the README with a brief description just to let folks know it's here if they need it (and why they might need it).